lib/sync-settings.coffee

```diff
@@ -7,23 +7,84 @@ _ = require 'underscore-plus'
 [GitHubApi, PackageManager] = []
 
 # constants
+syncUri = 'sync-settings:/'
+
 DESCRIPTION = 'Atom configuration storage operated by http://atom.io/packages/sync-settings'
 REMOVE_KEYS = ["sync-settings"]
 
-module.exports =
+SyncSettings =
   config: require('./config.coffee')
 
   activate: ->
-    GitHubApi ?= require 'github'
-    PackageManager ?= require './package-manager'
-    atom.commands.add 'atom-workspace', "sync-settings:backup", => @backup()
-    atom.commands.add 'atom-workspace', "sync-settings:restore", => @restore()
-    atom.commands.add 'atom-workspace', "sync-settings:view-backup", => @viewBackup()
+    # speedup activation by async initializing
+    # further optimization can be achieved by minimizing this file
```

Comment:
I wouldn't keep this comment, move it to an issue, of prefix with `TODO:` or `FIXME:`.